packages/sdk-viem/src/createArbitrumClient.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Chain,
+  PublicClient,
+  WalletClient,
+  createPublicClient,
+  http,
+} from 'viem'
+import {
+  ArbitrumDepositActions,
+  ArbitrumParentWalletActions,
+  arbitrumParentClientActions,
+  arbitrumParentWalletActions,
+} from './actions'
+
+export type ArbitrumClients = {
+  parentPublicClient: PublicClient
+  childPublicClient: PublicClient & ArbitrumDepositActions
+  parentWalletClient: WalletClient & ArbitrumParentWalletActions
+  childWalletClient?: WalletClient
+}
+
+export type CreateArbitrumClientParams = {
+  parentChain: Chain
+  childChain: Chain
+  parentRpcUrl?: string
+  childRpcUrl?: string
+  parentWalletClient: WalletClient
+  childWalletClient?: WalletClient
+}
+
+export function createArbitrumClient({
+  parentChain,
+  childChain,
+  parentRpcUrl,
+  childRpcUrl,
+  parentWalletClient,
+  childWalletClient,
```

Comment:
I don't think we need the childWalletClient here

Reply:
Actually, I would not request any client here, and just create everything within the function

Reply:
If they want to keep control on their client, they can extend with actions on their own. I would expect `createArbitrumClient` to do every thing for me and create the clients not extend them. But it's not an issue.